include/lsst/afw/table/Exposure.h

```diff
@@ -126,11 +126,14 @@ class ExposureRecord : public BaseRecord {
 
 protected:
 
-    ExposureRecord(PTR(ExposureTable) const & table);
+    explicit ExposureRecord(PTR(ExposureTable) const & table);
 
     virtual void _assign(BaseRecord const & other);
 
 private:
+
+    friend class ExposureTable;
```

Comment:
I don't think we have a policy.  I've put the ones here in `private` because that's where most of them were already.  I don't have a strong opinion.